py-rattler/tests/unit/test_version.py

```diff
@@ -17,3 +18,26 @@ def test_version_dash_normalisation() -> None:
 
     with pytest.raises(Exception):
         Version("1-.0dev+3.4-")
+
+
+def test_bump() -> None:
```

Comment:
These tests seem a little redundant as you already have them as part of the doctests.

Reply:
Ok. Should I remove all of them?

Reply:
You can keep the exception tests but I dont think the rest are required.